test/spec/modules/freewheel-sspBidAdapter_spec.js

```diff
@@ -86,6 +86,43 @@ describe('freewheelSSP BidAdapter Test', () => {
       expect(request.url).to.contain(ENDPOINT);
       expect(request.method).to.equal('GET');
     });
+
+    it('should add usp consent to the request', () => {
+      let uspConsentString = '1FW-SSP-uspConsent-';
+      let bidderRequest = {};
+      bidderRequest.uspConsent = uspConsentString;
+
+      const request = spec.buildRequests(bidRequests, bidderRequest);
+      console.log(request.data);
```

Comment:
Can you remove this console.log statement please?

Reply:
Removed.